src/aws-cli/aws_zsh_completer.sh

```diff
@@ -0,0 +1,60 @@
+# Source this file to activate auto completion for zsh using the bash
```

Comment:
Also, let's move these scripts under a new `vendor-scripts` folder and add a README similar to https://github.com/devcontainers/features/blob/main/src/dotnet/scripts/vendor/README.md#important-note